src/DirectusProvider.tsx

```diff
@@ -22,12 +24,18 @@ export const DirectusContext = React.createContext<DirectusContextTypeGeneric<an
 export const DirectusProvider = <T extends TypeMap = TypeMap>({
   apiUrl,
   options,
+  autoLogin,
   children,
 }: DirectusProviderProps): JSX.Element => {
+  const [user, setUser] = React.useState<UserType | null>(null);
+  const [authState, setAuthState] = React.useState<AuthStates>('loading');
+
+  const directus = React.useMemo(() => new Directus<T>(apiUrl, options), [apiUrl, options]);
+
   const value = React.useMemo<DirectusContextType<T>>(
     () => ({
       apiUrl: apiUrl,
```

Comment:
I think there is no need to repeat apiUrl and directus here

Reply:
yes, you are correct.